zipline/finance/slippage.py

```diff
@@ -156,10 +168,10 @@ def __init__(self, spread=0.0):
         """
         self.spread = spread
 
-    def process_order(self, bar_data, price, volume, order):
-        return create_transaction(
-            order,
-            bar_data.current_dt,
+    def process_order(self, data, order):
+        price = data.current(order.asset, "price")
```

Comment:
Like above, I believe only `close` should only be used here, possibly with is `nan` check before returning the order.